yarn-project/aztec.js/src/utils/authwit.ts

```diff
@@ -1,24 +1,50 @@
 import { FunctionCall, PackedArguments } from '@aztec/circuit-types';
-import { AztecAddress, GeneratorIndex } from '@aztec/circuits.js';
+import { AztecAddress, Fr, GeneratorIndex } from '@aztec/circuits.js';
 import { pedersenHash } from '@aztec/foundation/crypto';
 
 // docs:start:authwit_computeAuthWitMessageHash
 /**
  * Compute an authentication witness message hash from a caller and a request
- * H(caller: AztecAddress, target: AztecAddress, selector: Field, args_hash: Field)
+ * H(target: AztecAddress, H(caller: AztecAddress, selector: Field, args_hash: Field))
```

Comment:
Could you put here an example of a typical target and caller params? I am always translating these to real values in my mind (e.g. what is a token what is account contract etc) and it would improve readability to just have it here.